spec.bs

```diff
@@ -1830,70 +1845,66 @@ null |winningComponentConfig|:
      [=leading bid info/component seller reporting result=] to |reportingResult|.
   1. Otherwise, set |leadingBidInfo|'s [=leading bid info/seller reporting result=] to
      |reportingResult|.
+  1. [=map/Remove=] |browserSignals|["`desirability`"].
   1. [=map/Remove=] |browserSignals|["`dataVersion`"].
   1. [=map/Remove=] |browserSignals|["`modifiedBid`"].
   1. [=map/Remove=] |browserSignals|["`topLevelSellerSignals`"].
+
+  Note: Remove fields specific to {{ReportResultBrowserSignals}} so that they are not passed to
+  [=report win=].

```

Comment:
A note should explain why we don't want to pass them to [=report win=]. You could also mention that they were set in the first place because `reportResult()` needs them in order to ....